saas/fsm/unsanctioned_spend/fsm-unsanctioned_spend.pt

```diff
@@ -0,0 +1,247 @@
+name "SaaS Manager - Unsanctioned Spend Alert"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will raise an alert when Flexera SaaS Manager identifies unsanctioned spend on SaaS applications. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/unsanctioned_spend/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
product doesn't want to use alert.  instead change **raise an alert** to **create an incident**.